heimdall2/templates/heimdall-statefulset.yaml

```diff
@@ -314,6 +330,22 @@ spec:
             - name: OIDC_EXTERNAL_GROUPS
               value: {{ .Values.oidcExternalGroups | quote }}
             {{- end }}
+            {{- if .Values.oidcUsesPKCES256 }}
+            - name: OIDC_USES_PKCE_S256
+              value: {{ .Values.oidcUsesPKCES256 | quote }}
+            {{- end }}
+            {{- if .Values.oidcUsesPKCEPlain }}
+            - name: OIDC_USES_PKCE_PLAIN
+              value: {{ .Values.oidcUsesPKCEPlain | quote }}
+            {{- end }}
+            {{- if .Values.oidcUseHttpsProxy }}
+            - name: OIDC_USE_HTTPS_PROXY
+              value: {{ .Values.oidcUseHttpsProxy | quote }}
+            {{- end }}
+            {{- if .Values.oidcHttpsProxy }}
+            - name: OIDC_USE_HTTPS_PROXY
```

Comment:
double OIDC_USE_HTTPS_PROXY but I think the other implementation has the better name